### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by hougland**
I think this was a super simple fix, maybe I missed something?

### CODE REVIEW

**Comment by kariabancroft** [comment]
The database files in the DB folder should not be included in your commit. Please delete and add to gitignore

File: spec/controllers/admin/categories_controller_spec.rb [hunk 1]
```diff
@@ -34,6 +34,14 @@
     end
   end
 
+  describe "test_new" do
```
Comment (kariabancroft) [inline spec/controllers/admin/categories_controller_spec.rb]: Do you need to test any other scenarios? Like validation errors? Here or in the cucumber scenarios?

**Comment by hougland** [comment]
Thanks Kari! Got the database files out of github, and added more tests for adding new and editing categories. I didn't do validation errors since I was focusing on testing the controller not model, but I can if you think that's helpful.

**Comment by kariabancroft** [comment]
Looks good to close!

**Pull request closed without merging**